plugins-base/XSFeatureNTP.py

```diff
@@ -104,15 +297,23 @@ def HandleKeyADD(self, inKey):
         if inKey == 'KEY_ENTER':
             inputValues = pane.GetFieldValues()
             Layout.Inst().PopDialogue()
+            Layout.Inst().TransientBanner(Lang("Adding %s NTP Server..." % inputValues['name']))
             try:
                 IPUtils.AssertValidNetworkName(inputValues['name'])
+
                 data=Data.Inst()
+                if data.ntp.method("") == "Default":
+                    data.NTPServersSet([server for server in data.ntp.servers([]) if "centos.pool.ntp.org" not in server])
```

Comment:
Can this be a list of constants like it is done here? https://github.com/xenserver/host-installer/commit/09f0562e3456bc7065908c38c9716612731f22bd